app/src/client/components/buildPage/FormField.tsx

```diff
@@ -6,6 +6,26 @@ import Select, { StylesConfig } from 'react-select';
 import { TextInput } from '../form/TextInput';
 import { SECRETS_TO_MASK } from '../../utils/constants';
 import { TextArea } from '../form/TextArea';
+import { SelectOption } from './PropertySchemaParser';
+
+const customStyles: StylesConfig<SelectOption, false> = {
```

Comment:
If I understand correctly this style is used to add `+`  symbol to the option used to add new item. (and keep style as is for other options)

Since it takes some time to figure this out, maybe instead `customStyles` use more specific name, perhaps: `markAddPropertyOption`

Reply:
In the future, maybe it could be considered to have different Option components, one for regular option, and another for `AddPropertyOption,` because they not only have different visual representation, they are going to also behave quite differently. By using different components in the start we would not have to check what kind of option it is in onCiick handler.

(Css trick is interesting nevertheless )